controllers/endpoint/endpoints_controller.go

```diff
@@ -26,12 +26,18 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-// EndpointReconciler reconciles an Endpoints object
-type EndpointReconciler struct {
+const (
+	controllerName = "endpoint"
+)
```

Comment:
Nit:
```suggestion
const controllerName = "endpoint"
```

Reply:
its done in both the controllers now. PTAL